src/areaSearch/areaSearchApplicationSuccessPage.tsx

```diff
@@ -15,17 +15,31 @@ import { initializeAreaSearchForm } from './helpers';
 
 export interface Props {
   initializeForm: typeof initialize;
+  applicationIdentifiers: string;
 }
 
 const AreaSearchApplicationSuccessPage = ({
   initializeForm,
+  applicationIdentifiers,
 }: Props): JSX.Element => {
   const { t } = useTranslation();
 
   useEffect(() => {
     initializeForm(AREA_SEARCH_FORM_NAME, initializeAreaSearchForm());
   }, []);
 
+  let successMessage = t(
```

Comment:
An idea to avoid using let, make a helper function `getSuccessMessage`, and use it directly in the render part.